cli/add.go

```diff
@@ -27,7 +27,7 @@ func ConfigureAddCommand(app *kingpin.Application, a *AwsVault) {
 		Required().
 		StringVar(&input.ProfileName)
 
-	cmd.Flag("env", "Read the credentials from the environment (AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY)").
+	cmd.Flag("env", "Read the credentials from the environment (AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY & AWS_SESSION_TOKEN)").
```

Comment:
```suggestion
	cmd.Flag("env", "Read the credentials from the environment (AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and AWS_SESSION_TOKEN)").
```
Using same wording as here https://github.com/99designs/aws-vault/blob/master/USAGE.md

Reply:
Done, thanks for the idea!